### DISCUSSION TITLE
How to use make parameter instead of general.h

### DISCUSSION THREAD

**Discussion by kevin-olbrich** [ipxe/ipxe#652] [Q&A] [answered]
Hello,

I regularly build the master branch with console and framebuffer support. Is it possible to set the "defines" via a make parameter instead of the "general.h"?

I tried it with CFLAGS and EXTRA_CFLAGS but that only produces broken binaries.

Thank you!

**Comment by mcb30** [comment, marked as answer]
No, you can't change this via a make parameter, sorry.  You might find it easiest to edit `config/local/general.h` to override the parameters you need: those files won't be modified when you pull a newer version of the source tree.
